test/scripts/filters/external_link.js

```diff
@@ -25,15 +25,15 @@ describe('External link', () => {
     const content = [
       '# External link test',
       '1. External link',
-      '<a href="http://hexo.io/">Hexo</a>',
+      '<a href="https://hexo.io/">Hexo</a>',
       '2. Internal link',
       '<a href="/archives/foo.html">Link</a>',
       '3. Ignore links have "target" attribute',
-      '<a href="http://hexo.io/" target="_blank">Hexo</a>',
+      '<a href="https://hexo.io/" target="_blank">Hexo</a>',
       '4. Ignore links don\'t have "href" attribute',
       '<a>Anchor</a>',
       '5. Ignore links whose hostname is same as config',
-      '<a href="http://maji.moe">moe</a>'
+      '<a href="https://maji.moe">moe</a>'
```

Comment:
Should change these fixture website into `example.com`.

Reply:
Thanks.
Fix all "maji.moe" domain to "example.com"